src/patternfly/components/InputGroup/examples/input-group-example.hbs

```diff
@@ -1,45 +1,45 @@
 {{#> input-group }}
-  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
-  {{#> form-controls controlType="textarea" name="textarea1" id="textarea1" }}
+  {{#> button btnClass="pf-m-secondary" id="textAreaButton1"}} Button {{/button}}
+  {{#> form-controls controlType="textarea" name="textarea1" id="textarea1" aria-label="textarea with buttons" aria-describedby="textAreaButton1" }}
 {{/form-controls}}
     {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group }}
-  {{#> form-controls controlType="textarea" name="textarea2" id="textarea2" }}
+  {{#> form-controls controlType="textarea" name="textarea2" id="textarea2" aria-label="textarea with button" aria-describedby="textAreaButton2" }}
 {{/form-controls}}
-    {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
+    {{#> button btnClass="pf-m-tertiary" id="textAreaButton2"}} Button {{/button}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group }}
-  {{#> dropdown id="dropdown-example-collapsed"}}
+  {{#> dropdown }}
     Dropdown
     <i class="fas fa-caret-down pf-c-dropdown__toggle-icon"></i> 
   {{/dropdown}}
-  {{#> form-controls controlType="input" input="true" type="text" id="textInput3" name="textInput3" }}
+  {{#> form-controls controlType="input" input="true" type="text" id="textInput3" name="textInput3" aria-label="input with dropdown and button" aria-describedby="inputDropdownButton1"}}
 {{/form-controls}}
-    {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
+    {{#> button btnClass="pf-m-secondary" id="inputDropdownButton1"}} Button {{/button}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group }}
-  {{#> form-controls controlType="input" input="true" type="text" id="textInput4" name="textInput4" }}
+  {{#> form-controls controlType="input" input="true" type="text" id="textInput4" name="textInput4" aria-label="another input with dropdown and button" aria-describedby="inputDropdownButton2" }}
   {{/form-controls}} 
-  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
-  {{#> dropdown id="dropdown-example-collapsed"}}
+  {{#> button btnClass="pf-m-secondary" id="inputDropdownButton2"}} Button {{/button}}
+  {{#> dropdown }}
```

Comment:
same as above - creates an empty `id` and `aria-labelledby`

Reply:
👍